### PR TITLE
chore: remove CI Score badge

### PR DESCRIPTION
**Pull request by choeqq**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.
- [x] docs change  

## Motivation

- it closes an issue: https://github.com/facebook/docusaurus/issues/9465

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

- https://github.com/facebook/docusaurus/issues/9465

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
Could you revert the whitespace changes?

**Comment by choeqq** [comment]
@Josh-Cena sure thing - done ✅

File: README.md
Comment (Josh-Cena) [inline README.md:19]:
```suggestion
  <a href="https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Ffacebook%2Fdocusaurus%2Ftree%2Fmain%2Fexamples%2Fclassic&project-name=my-docusaurus-site&repo-name=my-docusaurus-site"><img src="https://vercel.com/button" alt="Deploy with Vercel"/></a>
  <a href="https://app.netlify.com/start/deploy?repository=https://github.com/slorber/docusaurus-starter"><img src="https://www.netlify.com/img/deploy/button.svg" alt="Deploy to Netlify"></a>
```

**Review by Josh-Cena — COMMENTED** [review]
Please sign the CLA, so we may merge.

**Comment by choeqq** [comment]
@Josh-Cena done, waiting for the approval 👀

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**
